pkg/label/label.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2019 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package label
+
+import (
+	"crypto/sha256"
+	"fmt"
+)
+
+// MaxLength stores the max label length value supported by kubernetes.
+const MaxLength = 63
```

Comment:
I wonder if we should delete this and use a call to here instead: https://github.com/kubernetes/apimachinery/blob/master/pkg/util/validation/validation.go#L156

Reply:
Sure, I will update.